src/schema/v2/me/artistRecommendations.ts

```diff
@@ -0,0 +1,84 @@
+import { GraphQLFieldConfig, GraphQLInt } from "graphql"
+import { connectionFromArraySlice } from "graphql-relay"
+import { convertConnectionArgsToGravityArgs } from "lib/helpers"
+import { CursorPageable, pageable } from "relay-cursor-paging"
+import { createPageCursors } from "schema/v1/fields/pagination"
+import { ResolverContext } from "types/graphql"
+import gql from "lib/gql"
+import { artistConnection } from "schema/v2/artist"
+import { find } from "lodash"
+
+const MAX_ARTISTS = 50
+
+export const ArtistRecommendations: GraphQLFieldConfig<
+  void,
+  ResolverContext
+> = {
+  description: "A connection of artist recommendations for the current user.",
+  type: artistConnection.connectionType,
+  args: pageable({
+    page: { type: GraphQLInt },
+  }),
+  resolve: async (
+    _root,
+    args: CursorPageable,
+    { vortexGraphqlLoader, artistsLoader }
+  ) => {
+    if (!vortexGraphqlLoader || !artistsLoader) return
+
+    const { page, size, offset } = convertConnectionArgsToGravityArgs(args)
+
+    // Fetch artist IDs from Vortex
+
+    const vortexResult = await vortexGraphqlLoader({
+      query: gql`
+        query artistRecommendationsQuery {
+          artistRecommendations(first: ${MAX_ARTISTS}) {
```

Comment:
**thought/issue:** would it be possible to set an `offset` here and have `first` to be equal to `size` as we get it from the client. That way we will avoid making the same query when the user is trying to get to the second page and it will also fix the issue that we will only be having a maximum of 50 `MAX_ARTISTS` recommended artists

Reply:
the query offset would be then based on the `recommendedArtists` and not the artists themselves

Reply:
Oh yes, we can actually just forward the pagination arguments from the query to **Vortex**. I'll change that.